template/package.json

```diff
@@ -9,13 +9,15 @@
   },
   "dependencies": {
     "core-js": "^3.3.1",
+    "eslint-config-tram-one": "^3.0.0",
     "jest": "^24.8.0",
     "jest-transform-stub": "^2.0.0",
     "parcel-bundler": "^1.6.1",
     "regenerator-runtime": "^0.13.3",
     "sass": "^1.20.1",
     "tram-blueprints": "^1.0.0",
     "tram-one": "^8.1.1",
+    "parcel-plugin-nuke-dist": "^1.0.0",
```

Comment:
This still looks like it's in the wrong spot... Shouldn't this be with the other 'p' dependencies?

Reply:
sorry, it's fixed